docs/user-manual/xr/ar/hit-testing.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Hit Testing
+sidebar_position: 1
+---
+
+In the AR/MR context, the interaction of virtual world objects with the real world is achieved by many techniques, Hit Testing - allows one to pick a point in space by probing real-world geometry that is estimated by underlying AR systems.
+
+This can be used in various ways and one of the most common - placing a virtual object in space that feels planted in the real world.
```

Comment:
```suggestion
This can be used in various ways and one of the most common is placing a virtual object in space so that it feels planted in the real world.
```